src/tools/w3dviewer/meshproppage.cpp

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D View mesh property page
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "meshproppage.h"
+#include "assetmgr.h"
+#include "mesh.h"
+#include "meshmdl.h"
+#include "resource.h"
+#include "utils.h"
+#include "w3d_file.h"
+#include "w3dviewdoc.h"
+
+IMPLEMENT_DYNCREATE(CMeshPropPage, CPropertyPage)
+
+// clang-format off
+BEGIN_MESSAGE_MAP(CMeshPropPage, CPropertyPage)
+    ON_WM_CLOSE()
+END_MESSAGE_MAP()
+// clang-format on
+
+CMeshPropPage::CMeshPropPage(CString &name) : CPropertyPage(IDD_MESH, 0), m_name(name) {}
```

Comment:
`const CString &name`
because name is not modified

Reply:
fixed.